### ISSUE TITLE
Page language has not been identified (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#132]
3,811 occurrences of this issue but the fix is likely in one place. Every HTML root element needs the language attribute: 
`<html lang="en"></html>`

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [x] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@tubesoft : At the Newton meeting today, Bill said you would be working on these accessibility issues. I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

Could you have a look at this issue and confirm it's something you can fix? Once you confirm, we can figure out a testing workflow.

**Comment by tubesoft** [comment]
SiteImprove seems to indicate that each page of the edited texts needs `@lang="en"`.
I am editing `p5-to-html.xsl` like below:
<img width="1141" alt="image" src="https://github.com/IUBLibTech/newton_chymistry/assets/17519127/4e503b2f-f3c4-4bb1-a937-38d9e83688dd">

**Comment by mdalmau** [comment]
@tubesoft yeah I figured this one was easy. 

How do you want to handle previous fixes? It’ll be good to crate a pull request for all the rendition issues, test and deploy before you start making the accessibility changes.

**Comment by tubesoft** [comment]
@mdalmau I just issues a pull request. See #144.

**Comment by mdalmau** [comment]
Since this issue is now part of https://github.com/IUBLibTech/newton_chymistry/pull/146, I will mark this issue as "done" so I can better see the remaining accessibility issues!
